gendoc/gendoc.go

```diff
@@ -23,7 +34,9 @@ func GenerateDocs(dt deftree.Deftree) map[string]io.Reader {
 	}
 
 	files := make(map[string]io.Reader)
-	files[dt.GetName()+"-service/docs/docs.md"] = strings.NewReader(response)
+	md := dt.(*deftree.MicroserviceDefinition)
+	svcname := strings.ToLower(findServiceName(md))
```

Comment:
Does this deal with:

```
syntax = "proto3";
service foo_bar {
```
Will the underscore still be here? As I believe that `package parsesvcname` will not have the `_` because of protoc-gen-go's `CamelCase` function.